design-documents/graph-ql/coverage/returns.graphqls

```diff
@@ -0,0 +1,197 @@
+type Query{
+    # See https://github.com/magento/architecture/blob/master/design-documents/graph-ql/framework/attributes-metadata.md
+    pageSpecificCustomAttributes(
+        page_type: CustomAttributesPageEnum!
+    ): CustomAttributeMetadata
+}
+
+type Mutation {
+    createReturn(input: CreateReturnInput!): CreateReturnOutput @doc(description: "Create a new return.")
+    addReturnComment(input: AddReturnCommentInput!): AddReturnCommentOutput @doc(description: "Add a comment to an existing return.")
+    addReturnTracking(input: AddReturnTrackingInput!): AddReturnTrackingOutput @doc(description: "Add tracking information to the return.")
+    removeReturnTracking(input: RemoveReturnTrackingInput!): RemoveReturnTrackingOutput @doc(description: "Remove tracking information from the return.")
+}
+
+input CreateReturnInput {
+    items: [CreateReturnItemInput]!
+    comment_text: String
+}
+
+input CreateReturnItemInput {
+    order_item_id: ID!
+    quantity_to_return: Float!
+    selected_custom_attributes: [ID!]
+    entered_custom_attributes: [EnteredCustomAttributeInput!]
+}
+
+input EnteredCustomAttributeInput {
+    id: ID!
+    value: String!
+}
+
+type CreateReturnOutput {
+    return: Return
```

Comment:
In Luma today, after you create a return, you're taken back to the `My Returns` view. It would be really nice if we could have a `returns: [Return]` here so the UI has the option to grab that view's data without multiple, dependent round trips

Reply:
Basically asking for the same field from `Customer` to also be here:

```graphql
type CreateReturnOutput {
    return: Return
    returns(
        pageSize: Int = 20 @doc(description: "Specifies the maximum number of results to return at once. Defaults to 20."),
        currentPage: Int = 1 @doc(description: "Specifies which page of results to return. The default value is 1."),
    ): Returns @doc(description: "Information about the customer returns.")
}
```

Reply:
This will complicate server-side implementation, but may make sense from client perspective. I saw you followed the same approach in negotiable quote schema. 
@nrkapoor what do you think?

Reply:
@DrewML @paliarush I agree with Andrew. It will provide more flexibility.

Reply:
> This will complicate server-side implementation

I'm a bit curious to hear more about the complications here. Is it that the work is harder to do, or concerns around load on the server?

Reply:
Done.

@DrewML Extra resolvers should be implemented.